package/boot/uboot-envtools/files/ramips

```diff
@@ -39,6 +39,7 @@ ravpower,rp-wd03)
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x4000" "0x1000" "0x1000"
 	;;
+beeline,smartbox-turbo-plus|\
```

Comment:
Please move block due to alphabetic sorting (of the first entries).

Reply:
fixed